forms-flow-components/src/components/CustomComponents/HistoryModal.tsx

```diff
@@ -242,6 +246,7 @@ export const HistoryModal: React.FC<HistoryModalProps> = React.memo(
                 <RevertField
                   variant="secondary"
                   size="sm"
+                  disabled={currentVersionId == entry.id}
```

Comment:
Here , disabling should be based on whether the type is BPMN or NOCODE . It should be disabled if the type is BPMN @shuhaib-aot